RAGAgent/tools/tools.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import os
+
+import requests
+
+
+def opea_rag_query(query):
+    ip_address = os.environ.get("ip_address")
+    url = f"http://{ip_address}:8889/v1/retrievaltool"
+    content = json.dumps({"text": query})
+    print(url, content)
```

Comment:
shall we use LOG_ERROR, LOG_VERBOSE like micro to control the print level? I didn't think this is needed for normal running env.